syntax = "proto2";

message UpdateMessage
{
    required bool   wants_update  = 1;
    required uint32 version_major = 2;
    required uint32 version_minor = 3;
    required uint32 version_patch = 4;
    required uint32 game_name     = 5;
    required uint32 host_platform = 6;
}
